db/agents.yml

```diff
@@ -315,6 +315,14 @@ agents:
     name: uTorrent
     type: Desktop App # and mobile???
     os: null
+  - regex: ^WNYC.+Android
+    name: WNYC App
```

Comment:
Yes, and this is consistent with other station apps, which are "[call letters] App"